binrz/rz-test/load.c

```diff
@@ -612,7 +612,10 @@ static bool database_load(RzTestDatabase *db, const char *path, int depth) {
 				eprintf("Skipping %s" RZ_SYS_DIR "%s because it requires additional dependencies.\n", path, subname);
 				continue;
 			}
-			if ((!strcmp(path, "archos") || rz_str_endswith(path, RZ_SYS_DIR "archos")) && strcmp(subname, RZ_TEST_ARCH_OS)) {
+			if ((!strcmp(path, "archos") || rz_str_endswith(path, RZ_SYS_DIR "archos")) && strcmp(subname, RZ_TEST_ARCH_OS) &&
+				!(!strcmp(subname, "not-windows-x64") &&
+					!rz_str_startswith(RZ_TEST_ARCH_OS, "windows-") &&
+					rz_str_endswith(RZ_TEST_ARCH_OS, "-x64"))) {
```

Comment:
do we really need the specific reference to windows here? Can't we rewrite it with something like `startswith(not-) && strstr(RZ_TEST_ARCH_OS, second-part-of-the-path)`?

Reply:
no, yes 3794dbd2d457c240297aff94468b3e85fa7d5f89